domains/szaby.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "sz4by",
+    "email": "xat.king6969@gmail.com"
+  },
+  "records": {
+    "CNAME": "weboldalam.onrender.com"
```

ANAME is currently not supported records, I recommend it to change in A records instead
```suggestion
    "A": ["216.24.57.1"]
```